packages/okta-react/README.md

```diff
@@ -110,15 +110,27 @@ export default withAuth(class Home extends Component {
     }
   }
 
-  componentDidUpdate() {
+  async login() {
```

Comment:
We need to make sure these methods are bound in the constructor.

```javascript
constructor(props) {
    super(props);
    this.state = { authenticated: null };
    this.checkAuthentication = this.checkAuthentication.bind(this);
    this.login = this.login.bind(this);
    ...
    this.checkAuthentication();
  }

```

Reply:
👍 good catch.  I've also removed the last line in the constructor, `this.checkAuthentication();`, as that's not needed (we should be calling that only on the component lifecycle methods, as we do below).  JFYI that I rebased this branch against master and force pushed